src/app/components/Alert/Alert.tsx

```diff
@@ -9,31 +9,60 @@ type AlertProps = {
 	size?: "small" | "default" | "large";
 };
 
-const Wrapper = styled.div(getStyles);
+const AlertContent = styled.div<{ size: string }>`
+	${({ size }) => {
+		switch (size) {
+			case "small":
+				return tw`px-6 py-4`;
+			case "default":
+				return tw`px-8 py-6`;
+			case "large":
+				return tw`px-10 py-8`;
+		}
+	}}
+`;
+
+const AlertIconWrapper = styled.div<{ size: string }>`
+	${({ size }) => {
+		switch (size) {
+			case "small":
+				return tw`px-4 py-2`;
+			case "default":
```

Comment:
Could simply use `default:` instead of `case "default":` to avoid that the default needs to be explicitly specified as a size.